core/deployment/src/main/java/io/quarkus/deployment/builditem/AdditionalApplicationArchiveBuildItem.java

```diff
@@ -1,21 +1,20 @@
 package io.quarkus.deployment.builditem;
 
-import java.nio.file.Path;
-
+import io.quarkus.bootstrap.model.PathsCollection;
 import io.quarkus.builder.item.MultiBuildItem;
 
 /**
  * An additional application archive
  */
 public final class AdditionalApplicationArchiveBuildItem extends MultiBuildItem {
 
-    private final Path path;
+    private final PathsCollection path;
 
-    public AdditionalApplicationArchiveBuildItem(Path path) {
+    public AdditionalApplicationArchiveBuildItem(PathsCollection path) {
         this.path = path;
     }
 
-    public Path getPath() {
+    public PathsCollection getPath() {
```

Comment:
This may break external extensions.

Reply:
I changed it to getPaths and added a deprecated method